tests/integration/test_issue_1179.py

```diff
@@ -21,11 +21,14 @@ def test_pip_2020_resolver_engaged():
 
     # The Pip legacy resolver cannot solve the following requirements but the 2020 resolver can.
     # Use this fact to prove we're plumbing Pip resolver version arguments correctly.
-    pex_args = ["boto3==1.15.6", "botocore>1.17<1.18.7", "--", "-c", "import boto3"]
+    pex_args = ["boto3==1.15.6", "botocore>1.17,<1.20", "--", "-c", "import boto3"]
```

Comment:
Modern packaging switched from pyparsing to a be-spoke parser and fixed the parsing bug that allowed `botocore>1.17<1.18.7` previously (missing comma / AND). The `<1.18.7` -> `<1.20` is to force a naive conflict that pip-legacy-resolver can't handle - the actual point of the test. That version, 1.19.63, is now encoded in the test assertions.